src/drivers/spektrum_rc/spektrum_rc.cpp

```diff
@@ -50,8 +50,11 @@
 #include <uORB/topics/input_rc.h>
 
 // Snapdraogon: use J12 (next to J13, power module side)
```

Comment:
We have been using tty-1 for RC.
It was "rc_receiver start -D /dev/tty-1" when we used rc driver from fc_addon.